MOBILE_API.xml

```diff
@@ -8133,6 +8167,9 @@
         <param name="myKey" type="MyKey" mandatory="false">
             <description>Information related to the MyKey feature</description>
         </param>
+        <param name="windowStatus" type="WindowStatus" array="true" minsize="0" maxsize="100" mandatory="false">
```

Comment:
@IGapchuk add the since 6.2 tag

Reply:
@IGapchuk  fixed in https://github.com/smartdevicelink/rpc_spec/pull/250/commits/809816e52db9c0f9b7485dcee554fa2b9efdfd5e